packages/cli/lib/main.js

```diff
@@ -0,0 +1,37 @@
+import { defineCommand, runMain, showUsage } from 'citty'
+import { colors } from 'consola/utils'
+import path from 'node:path'
+import { URL } from 'node:url'
+import { getPackageJSON } from './utils.js'
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname)
+const { version, description } = getPackageJSON(path.resolve(__dirname, '..'))
+
+/**
+ * NOTE:
+ *  currenctly, It's hard to customize usage with citty `renderUsage`.
```

Comment:
Please let me know or feel free to open an issue for what you like to be customizable or better rendered in citty usage renderer 👍🏼 :)